app/models/report.rb

```diff
@@ -63,4 +63,27 @@ def self.compute_employees_velocity(current_employee)
     end
     @employee_tasks_data
   end
+
+  def self.attendance_data(current_employee)
+    end_date = "31-12-#{Date.today.year}".to_date
+    week_days = [1, 2, 3, 4, 5]
+    attendances_array = []
+    current_employee.company.employees.each do |employee|
+      @attendances = employee.attendances.where('status = ?', Attendance::STATUS[:PRESENT]).order(login_time: :asc).select { |attendance| attendance.login_time.year == Date.today.year && attendance.logout_time != nil }
```

Comment:
can you do it like: ? 
`.where(status: Attendance::STATUS[:PRESENT])`

Reply:
Also, you could write a model function in attendance.rb and return @attendances from there